logging/logging.go

```diff
@@ -121,6 +121,13 @@ func (instance *Logging) GetInstanceLog(request cloudprotocol.RequestLog) error
 		return err
 	}
 
+	if len(logRequest.instanceIDs) == 0 {
+		log.WithField("logID", request.LogID).Error("No instance ids for log request")
```

Comment:
As we agreed that  it is not error, change it to debug.

Reply:
fiexed